### PR TITLE
Allow only ideal PTZ constraints in gUM

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR updates the PTZ explainer with the resolution "05" at https://www.w3.org/2020/09/15-webrtc-minutes.html#r05
I believe @eehakkin will follow with spec changes.

@jan-ivar @youennf @riju

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
(gentle ping)

File: ptz-explainer.md [hunk 1]
```diff
@@ -107,12 +100,11 @@ if ("tilt" in capabilities) {
...
+tilt, and zoom constraints. Using mandatory ones will cause the returned promise
+to reject with `OverConstrainedError`.
 
 ```js
-// User is prompted to grant both camera and PTZ access in a single call.
-// If the camera does not support PTZ or user denies PTZ permission, it fails
-// as PTZ constraints are required.
 const videoStream = await navigator.mediaDevices.getUserMedia({
   // [NEW] Website asks to reset camera pan.
   video: { pan: 1 },
```
Comment (youennf) [inline ptz-explainer.md:110]: Why 1 and not 0?

**Review by youennf — APPROVED** [review]

File: ptz-explainer.md (hunk 1 above)
Comment (beaufortfrancois) [reply ptz-explainer.md:110]: You're right. It will mostly be `0`, not `1`

Comment (beaufortfrancois) [reply ptz-explainer.md]: Fixed in https://github.com/w3c/mediacapture-image/pull/258/commits/05174085a0f6affc5b3b3cb6bb27b01ca611e19a. Thanks!

**Review by riju — APPROVED** [review]
LGTM :)

**Pull request merged**